app/src/main/java/com/infomaniak/drive/ui/fileList/SelectFolderActivity.kt

```diff
@@ -57,26 +57,34 @@ class SelectFolderActivity : BaseActivity() {
         val customArgs = navigationArgs.customArgs
         val currentFolderId = navigationArgs.folderId.getIntOrNull()
         val disabledFolderId = navigationArgs.disabledFolderId.getIntOrNull()
-        val currentUserDrive = UserDrive(userId, driveId)
+        DriveInfosController.getDrive(driveId = driveId, maintenance = false)?.let { currentDrive ->
```

Comment:
Make it in background thread with a liveData

Reply:
As we discussed, we'll keep that here since moving that to a background thread will break everything (See comment).